### PR TITLE
Py2to3

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
Upgraded django-name to python3. Please review the changes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     'python-dateutil==2.7.3',
     'markdown2==2.3.6',
     'djangorestframework==3.9.1',
-    'pynaco==0.1.0'
+    'pynco @ git+https://github.com/unt-libraries/pynaco@py3',
```
Comment (somexpert) [inline setup.py]: got a typo here: pynco -> pynaco

File: tests/test_name_detail.py [hunk 2]
```diff
@@ -21,9 +21,9 @@ def test_person_itemprops(client):
     response = client.get(
         reverse('name:detail', args=[name.name_id]))
 
-    assert 'itemprop=\"name\"' in response.content
-    assert 'itemprop=\'url\'' in response.content
-    assert 'itemprop=\"birthDate\"' in response.content
+    assert b'itemprop=\"name\"' in response.content
+    assert b'itemprop=\'url\'' in response.content
```
Comment (somexpert) [inline tests/test_name_detail.py:25]: This seems a little odd to me, is there a reason for the mixed quotes between this line and the next/previous?

Comment (somexpert) [reply tests/test_name_detail.py:25]: same comment for the other changes shown in this module

Comment (madhuBayy) [reply tests/test_name_detail.py:25]: It's the way it is designed in [name_detail.html](https://github.com/unt-libraries/django-name/blob/py2to3/name/templates/name/name_detail.html#L60) page.

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # vim: set ft=conf
 
-FROM python:2.7
+FROM python:3.7
 ENV PYTHONUNBUFFERED 1
 ENV PYTHONPATH /app
 RUN apt-get update -qq && apt-get install -y python-mysqldb default-mysql-client
```
Comment (ldko) [inline Dockerfile:6]: I needed to add  `python-dev` to packages that get installed for compiling `mysqlclient` and `psycopg2` to get the tests to run with `docker-compose run --rm web tox`

Comment (madhuBayy) [reply Dockerfile:6]: I see. I might have already installed `python-dev` so didn't have to install it for this project.

**Review by ldko — COMMENTED** [review]
These changes look good and work for me. I think we need to bump the version in `name/__init__.py`, perhaps to version 3.0.0 though. What do you think @madhulika95b @somexpert of that version? Also, Madhulika should be added to the README and docs/index.rst contributors.

**Comment by madhuBayy** [comment]
Yea...I think version 3.0.0 makes sense. I'll add my name to those docs too :)

**Comment by somexpert** [comment]
I don't have anything against a major version bump like that. django-name 3 for Python 3 :smile:

**Review by ldko — APPROVED** [review]

File: docs/index.rst [hunk 4]
```diff
@@ -37,4 +37,5 @@ Acknowledgements
 .. _`Lauren Ko`: http://github.com/ldko
 .. _`Mark Phillips`: http://github.com/vphill
 .. _`Gio Gottardi`: http://github.com/somexpert
+.. _`Madhulika Bayyavarapu`: http://github.com/madhulika95b
```
Comment (somexpert) [inline docs/index.rst]: May as well update all of these to https, yeah?

Comment (madhuBayy) [reply docs/index.rst]: ok. Doing that wouldn't hurt. I'll update all of them.

File: docs/index.rst [hunk 5]
```diff
@@ -32,9 +32,10 @@ Acknowledgements
 - `Mark Phillips`_
 - `Gio Gottardi`_
```
Comment (somexpert) [inline docs/index.rst:33]: oh, we gotta add you up here too!

Comment (madhuBayy) [reply docs/index.rst:33]: Right!!

**Review by somexpert — APPROVED** [review]

**Pull request merged**